src/components/Widget/inputs.tsx

```diff
@@ -37,18 +56,21 @@ export function useSyncWidgetInputs({
 
   const [type, setType] = useState<SwapValue['type']>(TradeType.EXACT_INPUT)
   const [amount, setAmount] = useState<SwapValue['amount']>(EMPTY_AMOUNT)
-  const [tokens, setTokens] = useState<SwapTokens>(defaultTokens)
+  const [tokens, setTokens] = useState<SwapTokens>({
+    ...defaultTokens,
+    [Field.OUTPUT]: defaultTokens[Field.OUTPUT] ?? defaultTokens.default,
+  })
```

Comment:
set the initial value here the classic way, rather than relying on the hook below